shared/elements/gaia_grid/js/grid_view.js

```diff
@@ -544,13 +544,13 @@
           item.setCoordinates(xPosition, this.layout.offsetY);
           if (!item.active) {
             item.render();
+          }
 
-            if (item.detail.type === 'divider') {
-              if (oddDivider) {
-                item.element.classList.add('odd');
-              } else {
-                item.element.classList.remove('odd');
-              }
+          if (item.detail.type === 'divider') {
```

Comment:
Sorry forget this, the previous condition is over items[i] so please keep you code